spec/models/mentee_application_state_spec.rb

```diff
@@ -24,40 +24,48 @@
 
 RSpec.describe MenteeApplicationState do
   describe '.next' do
-    context 'when the status is pending' do
-      let(:status) { :pending }
+    context 'when the status is application_received' do
+      let(:status) { :application_received }
```

Comment:
while I plan on ripping out these specs in the next PR, I want to make sure to update them so that the existing functionality is not changed.